### PR TITLE
refactor: footer

### LINKED ISSUE #1005: [Site]:  Add a Social media icons in footer
### Current State

**Description of the current state/issue**:
The APISIX website footer currently lacks a social component that lists the different social media icons for Twitter, LinkedIn, Youtube, and Medium.
**Link to the page**:
https://apisix.apache.org/
**Screenshots (if relevant)**:
Screenshot 2022-04-05 at 10 04 30

### Desired State

**Description of the change you are proposing**:
Add a Social component that will contain icons and links to these platforms

### Would you like to fix this issue?

No. Please assign another contributor

### PR DESCRIPTION
**Pull request by SkyeYoung**
Fixes: #1005

Changes:

1. Eject footer
2. Simplify component structure
3. Redesign style
4. Add social media icons

Screenshots of the change:

New:
image
image

Old:
image

### CODE REVIEW

File: website/src/css/util.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+$breakpoints: (
+    xs: 369px,
+    sm: 736px,
+    md: 1024px,
```
Comment (juzhiyuan) [inline website/src/css/util.scss]: Format with 2 spaces? others lgtm

File: website/src/css/util.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    @else {
+        @if (map-has-key($breakpoints, $lower)==false) {
+            @warn 'Lower breakpoint invalid: #{$lower}.';
+        }
+
+        @if (map-has-key($breakpoints, $upper)==false) {
+            @warn 'Upper breakpoint invalid: #{$upper}.';
+        }
+    }
+}
```
Comment (bzp2010) [inline website/src/css/util.scss]: EOL?

File: website/src/theme/Footer/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  max-width: 680px;
+  font-size: 12px;
+  color: $links-color;
+
+  margin-top: 2.4rem;
+
+  @include respond-below(sm) {
+    text-align: left;
+  }
+}
```
Comment (bzp2010) [inline website/src/theme/Footer/styles.module.scss]: EOL?

**Review by bzp2010 — DISMISSED** [review]

File: website/src/css/util.scss (hunk 1 above)
Comment (SkyeYoung) [reply website/src/css/util.scss]: done

File: website/src/css/util.scss (hunk 2 above)
Comment (SkyeYoung) [reply website/src/css/util.scss]: done

File: website/src/theme/Footer/styles.module.scss (hunk 3 above)
Comment (SkyeYoung) [reply website/src/theme/Footer/styles.module.scss]: done

**Review by SylviaBABY — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**